plugin/trino-elasticsearch/src/test/java/io/trino/plugin/elasticsearch/TestElasticsearch6ConnectorTest.java

```diff
@@ -23,10 +23,10 @@
 
 import static java.lang.String.format;
 
-public class TestElasticsearch6IntegrationSmokeTest
```

Comment:
@findepi Do you think it should be enough to use SmokeTest for the different variants similar to how we do it for MemSQL etc.

@findinpath I think you should instead change the ES7 tests to extend `BaseConnectorSmokeTest`. The smoke test is a light handed test which tests most functionality at a high level while the `BaseConnectorTest` is very thorough and long running.

Reply:
@hashhar  the requirement stated in https://github.com/trinodb/trino/issues/7109 is about replacing the smoke test class with  the thorough `BaseConnectorTest` classes. I've followed the requirements of the issue specified above (cc: @losipiuk )

Reply:
When issue was created we did not have `BaseConnectorSmokeTest` yet :) What @hashhar says makes total sense to me.

Reply:
> @findepi Do you think it should be enough to use SmokeTest for the different variants similar to how we do it for MemSQL etc.

totally!

Reply:
`TestElasticsearch7ConnectorSmokeTest` extends now `BaseConnectorSmokeTest`